### PR TITLE
33 create stats and battle screens

### PR DESCRIPTION
**Pull request by OddJerbb**

### CODE REVIEW

File: step-strikers-app/step-strikers-app/Menu Screens/StatsViewController.swift [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        createSettingsButton(x: 325, y: 800, width: 40, height: 40)
+        
+        // Create menu title label
+        _ = createImage(x:27, y:0, w:338, h:200, name:"Menu Title Board")
+        _ = createLabel(x:53, y:91, w:286, h:89, font:"iso8", size:45, text:"STATS", align:.center)
+        
+        // Create the display board and its contents
+        _ = createImage(x:0, y:219, w:393, h:374, name:"Display Board")
+        
+        // TODO: Pull stats data from Firebase and use as label text
```
Comment (alekhyaku) [inline step-strikers-app/step-strikers-app/Menu Screens/StatsViewController.swift:28]: so these are just stacked images and labels? not in a table view?

File: step-strikers-app/step-strikers-app/Menu Screens/StatsViewController.swift [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        _ = createLabel(x: 265, y: 411, w: 150, h: 75, font: "munro", size: 33, text: "__ min", align: .left)
+        
+        // Display steps info
+        _ = createImage(x: 25, y: 618, w: 75, h: 75, name: "brown boots")
+        // TODO: Pull steps info from firebase and display it here
+        _ = createLabel(x: 130, y: 624, w: 253, h: 41, font: "munro", size: 28, text: "__ steps until boost", align: .left)
+        _ = createLabel(x: 130, y: 653, w: 253, h: 41, font: "munro", size: 28, text: "__ taken today", align: .left)
+        
+        // TODO: Display the player's class pixel art below
+        _ = createImage(x: 140, y: 716, w: 112, h: 112, name: "Fighter")
```
Comment (alekhyaku) [inline step-strikers-app/step-strikers-app/Menu Screens/StatsViewController.swift:53]: use the global character and the get character class method to get the actual class. Also, I'll add the walking ones to add image assets sometime tomorrow so we can change these to the walking ver of the characters.

Comment (OddJerbb) [reply step-strikers-app/step-strikers-app/Menu Screens/StatsViewController.swift]: Added.

**Review by alekhyaku — APPROVED** [review]

**Review by alekhyaku — APPROVED** [review]

**Review by jcderry5 — APPROVED** [review]
All the data is already pulled over! LGTM

**Pull request merged**